docs/README.md

```diff
@@ -0,0 +1,41 @@
+# Website
+
+This website is built using [Docusaurus 2](https://v2.docusaurus.io/), a modern static website generator.
+
+### Installation
+
+```
+$ yarn
+```
+
+### Local Development
```

Comment:
To achieve that I had to:
- go to main dir and run yarn build
- then go to docs dir and run yarn start

Reply:
Yes, I need to update it